sdk/datafactory/Microsoft.Azure.Management.DataFactory/src/Generated/Models/SqlServerStoredProcedureActivity.cs

```diff
@@ -75,7 +75,7 @@ public SqlServerStoredProcedureActivity(string name, object storedProcedureName,
         /// parameters. Example: "{Parameter1: {value: "1", type: "int"}}".
         /// </summary>
         [JsonProperty(PropertyName = "typeProperties.storedProcedureParameters")]
-        public IDictionary<string, StoredProcedureParameter> StoredProcedureParameters { get; set; }
+        public object StoredProcedureParameters { get; set; }
```

Comment:
This is a breaking change, we need to double confirm this change, and if this is necessary, we need to bump the major version to 6.0.0